string camera_id
bool enable_publishing
int64 height
int64 width
int64 fps
---
bool success